python/semantic_kernel/connectors/ai/open_ai/utils.py

```diff
@@ -57,7 +118,7 @@ def get_function_calling_object(kernel: Kernel, filter: Dict[str, List[str]]) ->
                     "exclude_function": ["plugin1-function1", "plugin2-function2"],
                     }
                 will return all functions except plugin1-function1 and plugin2-function2.
-        caller_function_name: the name of the function that is calling the other functions.
+        form_tool_call: if True, the function will return a list of tool calls, otherwise a list of functions.
```

Comment:
from_tool_call?

Reply:
"form the tool call" -> "form_tool_call" not "from"

Reply:
I've updated this to `is_tool_call` to remove any confusion.